### PR TITLE
S3 functions

### PR DESCRIPTION
**Pull request by rtanoeiro**
This PR aims to add 2 functions to get/post data from an s3 bucket

### CODE REVIEW

File: config/dev.py [hunk 1]
```diff
@@ -4,6 +4,9 @@
 database = "platform_dev"
```
Comment (aletheavilla) [inline config/dev.py:4]: Can we turn this to just `config.py` instead of a whole folder?

Comment (aletheavilla) [inline config/dev.py:4]: Can we turn this to just `config.py` instead of a whole folder?

**Review by aletheavilla — CHANGES_REQUESTED** [review]

File: config/dev.py (hunk 1 above)
Comment (rtanoeiro) [reply config/dev.py:4]: Absolutely! Do you want to create dictionaries to hold dev, stage, prod values?

Comment (aletheavilla) [reply config/dev.py:4]: Nah. Let's leave it to the person if they create those or not.

Comment (rtanoeiro) [reply config/dev.py:4]: Oki, I'll write more stuff on the README then

Comment (aletheavilla) [reply config/dev.py:4]: Thanks!

**Pull request merged**